content/docs/guides/10-minute-tutorial.md

```diff
@@ -125,57 +125,29 @@ Open the project in IntelliJ IDEA:
 
 **With Gradle**
 
-You can use the following `build.gradle` file as an example template. Use your IDE
-to start a new Gradle-based project.
+The easiest way to create this sample Cucumber project using Gradle is to convert the above generated Maven archetype into a Gradle project.
```

Comment:
With #389 in mind I think we should rephrase:

```suggestion
One way to create this sample Cucumber project using Gradle is to convert the above generated Maven archetype into a Gradle project.
```

Reply:
Making language more inclusive